src/funcs.c

```diff
@@ -596,10 +596,12 @@ void list_show_favorites(char* str)
 			const int length = strlen(helper);
 			for (int i = 0; i <= length - 1; i++) helper[i] = tolower(helper[i]);
 
-			if (item_games->favorite == 1 && item_games->hidden != 1 && strstr(helper, (char *)str))
```

Comment:
When the start_with_favorites is 1, the **str** in this line is NULL (check line 800 which calls this method). So, this condition is always **false**.